packages/agent-toolkit/src/monday-graphql/queries.graphql.ts

```diff
@@ -428,3 +428,42 @@ export const fetchCustomActivity = gql`
     }
   }
 `;
+
+export const getNotifications = gql`
+  query getNotifications($cursor: ID, $limit: Int, $filterRead: Boolean, $since: ISO8601DateTime) {
+    notifications(cursor: $cursor, limit: $limit, filterRead: $filterRead, since: $since) {
+      id
+      text
+      title
+      read
+
+      creators {
+        id
+        name
+      }
+
+      update {
+        id
+        text_body
+      }
+
+      item {
+        id
+        name
+        created_at
+        creator {
+          id
+        }
+      }
+
+      board {
```

Comment:
why do you need board, item, and so on here? 
also, is it not working correctly with all API tool?